tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -723,7 +736,11 @@ protected void updateViewport() {
   protected void onScrollChanged( int l, int t, int oldl, int oldt ) {
     super.onScrollChanged( l, t, oldl, oldt );
     updateViewport();
-    requestThrottledRender();
+    if( mShouldRenderWhilePanning ) {
+      requestRender();
```

Comment:
+1 (since it's optional anyway)

back to +1 on the whole PR, will merge when i get a minute